src/chttpd/src/chttpd.erl

```diff
@@ -362,6 +362,8 @@ catch_error(HttpReq, error, decryption_failed) ->
     send_error(HttpReq, decryption_failed);
 catch_error(HttpReq, error, not_ciphertext) ->
     send_error(HttpReq, not_ciphertext);
+catch_error(HttpReq, error, {erlfdb_error, 2101}) ->
+    send_error(HttpReq, transcation_too_large);
```

Comment:
Sorry, just realized there's a typo here:
```suggestion
    send_error(HttpReq, transaction_too_large);
```

Reply:
Good catch, and fixed in 4f7caf7